src/bro/executive/ui_tars_7b.py

```diff
@@ -148,13 +147,14 @@ def act(self, goal: str, effort: Effort) -> str:
                 assert False, "Unreachable"
             _logger.debug("Response: %s", response)
             resp_msg = response.choices[0].message
-            resp_text = resp_msg.content.strip()
-            ctx.append({"role": resp_msg.role, "content": resp_text})
-            new_items, msg = self._process(resp_text)
-            ctx += new_items
-            if msg is not None:
-                _logger.debug(f"🤖 Final message: {msg}")
-                return msg
+            if resp_msg.content is not None:
```

Comment:
Please restore the original code without `if` and precede it with:

```python
assert resp_msg.content is not None
```

This is because the logic is ill-defined for the case when the content is None